### PR TITLE
Changelog for `v2.8.0rc1`

### PR DESCRIPTION
**Pull request by remi-kazeroni**
## Description

Changelog for `v2.8.0rc1`

Related to #1951 

Link to documentation: [changelog](https://esmvaltool--1952.org.readthedocs.build/projects/ESMValCore/en/1952/changelog.html)

***

## [Before you get started](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#getting-started)

- [x] [☝ Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do

## [Checklist](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#checklist-for-pull-requests)

It is the responsibility of the author to make sure the pull request is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.

[1]: https://docs.esmvaltool.org/en/latest/community/review.html#technical-review
[2]: https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review

- [x] [🧪][2] The new functionality is [relevant and scientifically sound](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#scientific-relevance)
- [x] [🛠][1] This pull request has a [descriptive title and labels](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-title-and-label)
- [x] [🛠][1] Code is written according to the [code quality guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#code-quality)
- [x] [🧪][2] and [🛠][1] [Documentation](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#documentation) is available
~~- [ ] [🛠][1] [Unit tests](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#tests) have been added~~
- [x] [🛠][1] Changes are [backward compatible](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#backward-compatibility)
~~- [ ] [🛠][1] Any changed [dependencies have been added or removed](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#dependencies) correctly~~
- [x] [🛠][1] The [list of authors](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#list-of-authors) is up to date
- [x] [🛠][1] All [checks below this pull request](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-checks) were successful

***

To help with the number pull requests:

- 🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValCore/pulls) in this repository

### CODE REVIEW

File: doc/changelog.rst
Comment (remi-kazeroni) [inline doc/changelog.rst:41]: @valeriupredoi do we need to list this PR?

File: doc/changelog.rst
Comment (remi-kazeroni) [inline doc/changelog.rst:78]: And this one, @valeriupredoi?

File: doc/changelog.rst
Comment (valeriupredoi) [inline doc/changelog.rst:15]: we should do the heyho tech-team, what you reckon it's a good highlight (one option per person) dance, man - like we did in other releases, I'll post that here

File: doc/changelog.rst
Comment (valeriupredoi) [reply doc/changelog.rst:40]: nope

File: doc/changelog.rst
Comment (valeriupredoi) [inline doc/changelog.rst:63]: Saskia Loosveldt -> @sloosvel you want your full name, that contains some 12 names?

File: doc/changelog.rst
Comment (valeriupredoi) [reply doc/changelog.rst:74]: nope as well

File: doc/changelog.rst
Comment (valeriupredoi) [inline doc/changelog.rst:94]: same here for Saskia

**Comment by valeriupredoi** [comment]
@ESMValGroup/technical-lead-development-team tis that time of the year when the Release Manager and their Little Helper will ask you to please nominate a Highlight for the upcoming release (one nominated PR per person, please, ideally try some variations from what's been nominated before you): for me it's simple - @bouweandela 's PR #1609 "Godzilla"

File: doc/changelog.rst
Comment (sloosvel) [reply doc/changelog.rst:63]: I don't have my name in my GH account, that is why it appears like this. Is that an issue? It's been like this since the beginning.

Comment (valeriupredoi) [reply doc/changelog.rst:63]: not at all - but it's not fair to you not to be listed by name, like everyone else, am wondering why we've not seen and corrected this until now

**Comment by remi-kazeroni** [comment]
> @ESMValGroup/technical-lead-development-team tis that time of the year when the Release Manager and their Little Helper will ask you to please nominate a Highlight for the upcoming release (one nominated PR per person, please, ideally try some variations from what's been nominated before you): for me it's simple - @bouweandela 's PR #1609 "Godzilla"

Thanks for starting this @valeriupredoi. I had planned to start asking once the last PR would have been merged. I reckon I could have anticipated a bit and started that earlier. 
Anyway, I was planning to open a proper discussion on this as we normally do and tag the whole development community as contributions to the Core are not restricted to the TLT. More on this very soon, please all wait a little bit for me to check things 😅

File: doc/changelog.rst
Comment (sloosvel) [reply doc/changelog.rst:63]: But that's on me for not setting my account. I truly don't mind, but if for formatting issues it bother you I can change it.

File: doc/changelog.rst
Comment (remi-kazeroni) [reply doc/changelog.rst:15]: Do we really need to have the highlight section ready to release rc1? I've just started asking around in https://github.com/ESMValGroup/ESMValCore/discussions/1954 I'd prefer to release rc1 without the highlight section and add them later, in an rc2 or if that is not needed, we can just cherry-pick from main. I've open an issue about v2.8 changelog so that this is not forgotten: #1952

File: doc/changelog.rst
Comment (remi-kazeroni) [reply doc/changelog.rst:62]: @sloosvel, that's really up to you. Looking at the changelog, I see other contributors listed with their github username only. so I guess we don't have a strict policy on this. You seem to be the most active contributor not having your name in your GH account. Anyway, not really related to this PR in particular

**Review by valeriupredoi — APPROVED** [review]
godspeed, v2.8.0rc1 :boat:

File: doc/changelog.rst
Comment (valeriupredoi) [reply doc/changelog.rst]: good tackling, man!

**Pull request merged**
